src/screens/HomeScreen.js

```diff
@@ -99,11 +99,96 @@ export const HomeScreen = ({ navigation, route }) => {
     }, 500);
   };
 
+  const flatListData = [
+    {
+      categoriesNews,
+      fetchPolicy,
+      isShow: showNews,
+      navigation,
+      query: QUERY_TYPES.NEWS_ITEMS,
+      queryVariables: { limit: 3, excludeDataProviderIds }
+    },
+    {
+      buttonTitle: buttonPointsOfInterestAndTours,
+      fetchPolicy: fetchPolicy,
+      isShow: showPointsOfInterestAndTours,
+      navigate: 'CATEGORIES_INDEX',
+      navigation,
+      query: QUERY_TYPES.POINTS_OF_INTEREST_AND_TOURS,
+      queryVariables: { limit: 10, orderPoi: 'RAND', orderTour: 'RAND' },
+      title: headlinePointsOfInterestAndTours
+    },
+    {
+      buttonTitle: buttonEvents,
+      fetchPolicy: fetchPolicy,
+      isShow: showEvents,
+      navigate: 'EVENT_RECORDS_INDEX',
+      navigation,
+      query: QUERY_TYPES.EVENT_RECORDS,
+      queryVariables: { limit: 3, order: 'listDate_ASC' },
+      showVolunteerEvents,
+      title: headlineEvents
+    }
+  ];
+
+  return (
+    <SafeAreaViewFlex>
+      <FlatList
+        data={flatListData}
+        ListHeaderComponent={
+          <>
+            <ConnectedImagesCarousel
+              alternateAspectRatio
+              navigation={navigation}
+              publicJsonFile="homeCarousel"
+              refreshTimeKey="publicJsonFile-homeCarousel"
+            />
+
+            <Widgets widgetConfigs={widgetConfigs} />
+          </>
+        }
+        ListFooterComponent={
+          route.params?.isDrawer && (
+            <>
+              <HomeService />
+              <About navigation={navigation} withHomeRefresh />
+            </>
+          )
+        }
+        refreshControl={
+          <RefreshControl
+            refreshing={refreshing}
+            onRefresh={refresh}
+            colors={[colors.accent]}
+            tintColor={colors.accent}
+          />
+        }
+        renderItem={(item) => renderItem(item)}
```

Comment:
this can be simplified as you only take the param and passes the same param => `renderItem={renderItem}`